pkg/common/common.go

```diff
@@ -80,6 +80,9 @@ func MergeStringMaps(mapA, mapB map[string]interface{}) map[string]interface{} {
 }
 
 func RoundToTwoDecimalPlaces(v float64) float64 {
+	if math.IsNaN(v) || math.IsInf(v, 0) {
```

Comment:
Maybe it's better to fix the underlying issue (incorrect values passed to the function) than masking it?
It's also unclear why we return 0 when value is `+Inf`. When speaking about MemoryUsagePercent it can be 100% and not 0, although I'm not sure how this happened.

Reply:
> Maybe it's better to fix the underlying issue (incorrect values passed to the function) than masking it?
Agree, lets figure out a better fix.

> It's also unclear why we return 0 when value is +Inf.
The input I saw, example above had both `MemoryUsagePercent:+Inf` and `MemoryUsagePercent:NaN`.

> When speaking about MemoryUsagePercent it can be 100% and not 0, although I'm not sure how this happened.

I explained the case in the first post. Let me add more detail.
If configured with `mem_monitoring = false`, then https://github.com/cloudradar-monitoring/cagent/blob/6f95682f7181bc284aa1deca7d72a265ad9caa12/pkg/monitoring/processes/processes.go#L68 will be called with memStat = nil, which leads to 

https://github.com/cloudradar-monitoring/cagent/blob/6f95682f7181bc284aa1deca7d72a265ad9caa12/pkg/monitoring/processes/processes.go#L77

with systemMemorySize = nil

Reply:
Dropped the old patch and sent a better fix.